kolibri/plugins/learn/assets/src/views/content-cards/content-carousel.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+
+  <section class="content-carousel">
+
+    <header v-if="header" class="content-carousel-header">
+      <h2 v-if="header"> {{header}} </h2>
+      <sub v-if="subheader"> {{subheader}} </sub>
```

Comment:
hm. [According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sub), `<sub>` is pretty symantically inconsequential; it's only about style:
>The HTML <sub> element defines a span of text that should be displayed, for typographic reasons, lower, and often smaller, than the main span of text.

Why are you opposed, again?